examples/glade-cmip6.json

```diff
@@ -0,0 +1,137 @@
+{
+  "stac_version": "0.9.0",
+  "stac_extensions": [
+    "collection-assets",
+    "https://github.com/NCAR/esm-collection-spec/tree/master/schema.json"
```

Comment:
@m-mohr, Is this going to be the final location of the schema or should I use something different? I realize that during our sprint, we agreed to use this as a placeholder.

Reply:
Should be a placeholder, It's up to you guys to figure out where you could host a schema permanently. Not sure whether you can come up with something like https://schemas.ncar.com/esm/v0.2/schema.json or so.